app/web/features/auth/Settings.tsx

```diff
@@ -39,42 +33,43 @@ export default function Settings() {
 
   return (
     <>
-      <HtmlMeta title={ACCOUNT_SETTINGS} />
-      <PageTitle>{ACCOUNT_SETTINGS}</PageTitle>
+      <HtmlMeta title={t("account_settings_page.title")} />
+      <PageTitle>{t("account_settings_page.title")}</PageTitle>
       <Section
         className={classes.section}
-        title={CHANGE_GENDER}
-        content={CHANGE_GENDER_CONTACT}
+        title={t("account_settings_page.gender_section.title")}
+        content={t("account_settings_page.gender_section.explanation")}
```

Comment:
It is probably not scope of this ticket but I saw that for example in `features/donations` we use "body" instead of "explanation".

Reply:
That's a good point, maybe we should have some convention on standardising how we name these things... but that can be tricky as it really depends on the context of the component.

For example with this, I'm trying to name all of the sections consistently with "title", "description" and "explanation"... except the gender and birthdate sections don't have a description. I guess the "explanation" could change to something else, so maybe could name it `description2` or something like that.

Reply:
But yes, I'll be looking at the donations one next too, so I'll change it there if we come to some agreement on what's best :)